src/App.js

```diff
@@ -11,6 +11,8 @@ import VerifyRegistrationView from './views/VerifyRegistrationView/index';
 import ArticlesView from './views/ArticlesListingView';
 import CreateArticle from './views/CreateArticleView';
 import ArticleView from './views/ArticleView';
+import ProfileView from './views/UserProfileView';
+import profileEditView from './views/UserProfileView/profileEditView';
```

Comment:
Place all your views inside the views folder don't nest them.
```/views/UserProfileView/profileEditView';``` 
**Correct format:**
```/views/UserProfileView/```
```/views/profileEditView/```

Reply:
Hello @jomasim,

I have successfully implemented moving the folder _profileEditView_ directly into the views.
I also renamed it to conform with our folder convention to **ProfileEditView**
Thank you 
Regards
Steven Ennis